### PR TITLE
Introduce to servlets

### PR DESCRIPTION
**Pull request by gigbat**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -23,8 +25,15 @@
             <artifactId>mysql-connector-java</artifactId>
             <version>8.0.17</version>
         </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <version>3.1.0</version>
```
Comment (vicire) [inline pom.xml]: we need not use alfa version of dependencies.

File: src/main/webapp/web.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    <servlet>
+        <servlet-name>index</servlet-name>
+        <servlet-class>internet.shop.controllers.IndexController</servlet-class>
+    </servlet>
+
+    <servlet-mapping>
+        <servlet-name>index</servlet-name>
+        <url-pattern>/index</url-pattern>
+    </servlet-mapping>
+</web-app>
```
Comment (vicire) [inline src/main/webapp/web.xml]: add empty line

**Review by vicire — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (gigbat) [reply pom.xml]: done

File: src/main/webapp/web.xml (hunk 2 above)
Comment (gigbat) [reply src/main/webapp/web.xml]: done

**Review by vicire — APPROVED** [review]

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**